packages/libs/src/services/solana/transactionHandler.ts

```diff
@@ -330,6 +328,9 @@ export class TransactionHandler {
 
     // Await for tx confirmation
     try {
+      if (!txid) {
```

Comment:
fwiw `txid` is just the first signature of a transaction, we don't really need to get it from `sendRawTransaction()` imo